src/pages/patterns/fluid-variants/index.mdx

```diff
@@ -0,0 +1,200 @@
+| Expressive moment | An expressive experience creates a desired impact or pause in an experience. They are often used when users are learning, exploring, and interacting through impactful imagery, layout, and long-form reading. Expressive elements may evoke more personality and emotion. |
+
+<Row>
+<Column colLg={8}>
+
+![Example of fixed and fluid](/images/Layout_overview_Fluid-base-unit.svg)
+
+</Column>
+</Row>
+
+### When to use fluid variants
+
+- Use in expressive or important moments, to better capture the user’s attention
+  and offer a rhythmic break to the productive experience.
+- Use within or attached to compound components.
+- Use in spacious layouts where the fluid variants have space to expand and
+  breath.
+
+### When not to use fluid variants
+
+- Don’t use when white space is needed between input components or in dense
+  layout where smaller components may be preferred.
+- Don’t use in hyper productive moments like a complex form.
+- Don’t use inside accordions or similarly sectioned containers where the edge
+  of the fluid component align to the same edge as the accordion dividers
+  creating hierarchical confusion.
+
+## Types of fluid components
+
+Not all components come with a fluid variant but each fluid instance has a
+default counterpart.
+
+### Fluid inputs
+
+Fluid inputs are an alternative input style with the label placed inside the
+input field, aligning with the user input text. All text is contained inside the
+field, unlike the default input where the label and helper text sit above and
+below, respectively, outside the field container. Fluid inputs only have one
+height size at 64px but have a fluid width that hugs one or more edges of its
+container. See the individual input components for more guidance on each fluid
+component.
+
+<Row>
+<Column colLg={8}>
+
+![Exampls of fluid components](/images/fluid-pattern-inputs-1.png)
+
+</Column>
+</Row>
+
+### Fluid button
+
+Button alignment determines whether the button is a fixed or fluid element
+within a layout. If it is fluid, its width is defined as a percentage (often
+50%) of the container’s width and will often be attached to the bottom of the
+containers. Fluid buttons cannot be embedded in the middle of a flow/layout.
+They must hug the (bottom) edge of a container. Use fixed buttons if additional
+options or actions follow below the primary flow and primary action.
+
+A fixed button’s width is calculated by the label’s length plus an additional
+64px to the right, with only the left side aligning to grid. Additionally, there
+is a hybrid scenario where a fixed styled button can span a designated number of
+columns on the responsive column grid giving it a fluid width. For more guidance
+on fixed versus fluid buttons, see the button component.
+
+### Fluid containers
+
+Though not a formal component, fluid and fixed containers are applied in various
```

Comment:
Though not a formal component, fluid and fixed containers are applied in various existing components. Fluid containers, like a modal, calculate their width based on column spans or percentages and will fluctuate in size. Fixed containers, like a side panel, maintain a static width, cannot be collapsed and can exist outside the responsive grid.